### PR TITLE
Feature/docker improvements

### PR DESCRIPTION
**Pull request by mbicknese**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,47 +1,35 @@
...
-    image: yoshz/symfony-docker-bootstrap-web
-    ports:
-    - "80:80"
-    volumes:
-    - "./server/nginx.conf:/etc/nginx/conf.d/default.conf:ro"
-    - "./web:/var/www/web:ro"
-    depends_on:
-    - app
+    db:
+        image: mariadb:latest
```
Comment (TijmenWierenga) [inline docker-compose.yml]: Cutting edge! Usually bad practise but doesn't really harm in development I guess.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,47 +1,35 @@
...
+            MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+            MYSQL_DATABASE: ${MYSQL_DATABASE}
+            MYSQL_USER: ${MYSQL_USER}
+            MYSQL_PASSWORD: ${MYSQL_PASSWORD}
+    php:
+        build:
+            context: .
+        volumes:
+            - .:/code
+        environment:
```
Comment (TijmenWierenga) [inline docker-compose.yml:17]: In order to organise your environment variables you may want to use an `env_file` here.

File: docker-compose.yml [hunk 3]
```diff
@@ -1,47 +1,35 @@
...
+            SYMFONY_DEBUG: ${SYMFONY_DEBUG}
+            APP_SECRET: ${APP_SECRET}
+            DB_DRIVER: ${DB_DRIVER}
+            DB_USER: ${MYSQL_USER}
+            DB_PASSWORD: ${MYSQL_PASSWORD}
+            DB_NAME: ${MYSQL_DATABASE}
+            DB_HOST: ${DB_HOST}
+        links:
+            - db
+    web:
```
Comment (TijmenWierenga) [inline docker-compose.yml]: Look into Traefik. It's so nice 😜

**Review by TijmenWierenga — APPROVED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (mbicknese) [reply docker-compose.yml]: Jep I'll certainly do that at a later stage. Should also prevent me from running into port collisions.

File: docker-compose.yml (hunk 2 above)
Comment (mbicknese) [reply docker-compose.yml:17]: Can you elaborate more on that?

File: docker-compose.yml [hunk 4]
```diff
@@ -1,47 +1,35 @@
...
+        environment:
+            SYMFONY_ENV: ${SYMFONY_ENV}
+            SYMFONY_DEBUG: ${SYMFONY_DEBUG}
+            APP_SECRET: ${APP_SECRET}
+            DB_DRIVER: ${DB_DRIVER}
+            DB_USER: ${MYSQL_USER}
+            DB_PASSWORD: ${MYSQL_PASSWORD}
+            DB_NAME: ${MYSQL_DATABASE}
+            DB_HOST: ${DB_HOST}
+        links:
```
Comment (kloki) [inline docker-compose.yml]: You could use env-file instead declaring all the variables seperatly

**Pull request merged**